src/PhappCommandBase.php

```diff
@@ -112,7 +118,11 @@ protected function getPhappEnviromentVariables($directory = './', $manifest = NU
     foreach ($finder as $file) {
       // Add dotenv vars.
       $dotenv = new Dotenv();
-      $env_vars = array_replace($env_vars, $dotenv->parse(file_get_contents($file->getPathname()), $file->getPathname()));
+      // Parse dotenv vars.
+      $parsed_env_vars = $dotenv->parse(file_get_contents($file->getPathname()), $file->getPathname());
+      // Populate them to make sure all of them are available for next parsing.
+      $dotenv->populate($parsed_env_vars);
```

Comment:
This should not be done. The method does return the variables instead of populating them on purpose. Else, this will create issues when you run some command on multiple sub-sites what will have different values for some environment varirables, like PHAPP_BASE_URL. Overwriting might not work or at least lead to subtle issues like old variables of previous sitse being still available if not overwritten. That we should avoid.